### ISSUE TITLE
skanlite does not start

### ISSUE THREAD

**Issue by emanueleperuffo** [NixOS/nixpkgs#77550]
## Issue description

Skanlite does not start from my GNOME Wayland session.

The error, when starting skanlite from the command line:
```
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

[1]    20892 abort (core dumped)  skanlite
```

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.8, NixOS, 19.09.1781.d245ff1bb9b (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-19.09.1781.d245ff1bb9b"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by veprbl** [comment]
This is #65399, it can be fixed by adding `wrapQtAppsHook`

**Comment by worldofpeace** [comment]
fixed in 4b20959f9980665d1024d9b3bb8da43a6f9f8fa4

**Comment by worldofpeace** [comment]
backported to 19.09 at ed422d8513c273550ba91c0073c38351e6d8011c
